### DISCUSSION TITLE
Promotional Work

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Bforartists#2342] [General]
We need to make Bforartists a bit better known. So we spread the news at various boards and pages. It's also time to write a Wikipedia article. This task here is about all this promotional work.

**Comment by ReinerBforartists** [comment]
So far ...

English pages:

https://forum.unity.com/threads/bforartists-0-9-4-released.495712/
http://polycount.com/discussion/184041/blender-fork-bforartists/p1?new=1
http://forums.cgsociety.org/showthread.php?f=59&t=1466944
https://www.gamedev.net/forums/topic/692162-blender-fork-bforartists-version-094-released/?tab=comments#comment-5356531
https://www.united3dartists.com/forum/viewtopic.php?f=13&t=5336
https://www.reddit.com/r/3DMA/comments/70obag/blender_fork_bforartists_version_094_released/

German pages:

http://www.3d-ring.de/3d/forum.php?post_id=33379
http://www.gimpforum.de/showthread.php?p=158904#post158904
https://zfx.info/viewtopic.php?f=10&t=3867&start=75
https://www.3d-board.de/index.php/Thread/32906-Blender-Fork-Bforartists-Version-0-9-4-erschienen/?s=b82c25dc27be3392b7a500e29cf41afd15aee12d

Contacted ZDNet, and waiting for response. Contacted CGPress, and waiting for response.

Download pages:

http://www.softpedia.com/get/Multimedia/Graphic/Graphic-Editors/Bforartists.shtml
http://softgui.de/softwares/bforartists
http://www.computerbild.de/download/Bforartists-19622603.html

**Comment by Davis-BfA** [comment]
I´ll drop this here, the logo needs to be updated on alternativeto. https://alternativeto.net/software/bforartists/

**Comment by Davis-BfA** [comment]
Here is a list of blogs and websites I contacted today to include BfA into their articles or software lists:
https://www.easyrender.com
https://all3dp.com
http://www.3ders.org
https://www.lifewire.com
https://windowsreport.com
https://www.sculpteo.com
https://www.hongkiat.com
https://blog.capterra.com
https://www.3dnatives.com
https://3dprinting.com
https://www.matterhackers.com
https://www.dobot.cc
http://3dprintingforbeginners.com
https://www.shapeways.com
https://en.99designs.de
https://www.digitaltrends.com
https://mashtips.com
http://innov8tiv.com
https://www.scan2cad.com
http://teknologya.com
https://www.pannam.com
http://www.viralhax.com
https://medium.com
https://help.sketchfab.com
https://ultimaker.com
https://makezine.com
https://www.topbestalternatives.com
https://ubernerd.com.au
https://www.opensourcealternative.org
https://pannam.com
https://cadalyst.com
https://dexigner.com
https://realspace3d.com
https://archicgi.com
https://tenlinks.com
https://architecturalrenderingoffice.com

**Comment by ReinerBforartists** [comment]
Good job!

**Comment by ReinerBforartists** [comment]
Good catch with the graphics. I will have a look to update the graphics at alternativeto tomorrow.

**Comment by ReinerBforartists** [comment]
Images at aternative.to updated. They are currently pending ...

**Comment by Davis-BfA** [comment]
We made it onto the Make: blog.
https://makezine.com/2018/03/23/painless-blender-interface/
I also got a comment approved on the mashtips list.
https://mashtips.com/3d-modeling-tools/

**Comment by ReinerBforartists** [comment]
Yay \o/

**Comment by ReinerBforartists** [comment]
We have a news at CG Channel and CG Press. 

http://cgpress.org/archives/bforartists-1-0-released.html
http://www.cgchannel.com/2018/05/bforartists-team-releases-bforartists-1-0/

**Comment by Draise14** [comment]
80.lvl also mentioned BFA. I'll have to find the link, it's on the facebook page.

**Comment by ryliejamesthomas** [comment]
I've mentioned it in my Game Making Tools Wiki ;3

https://www.gamemaking.tools/wiki/index.php/Bforartists

**Comment by ReinerBforartists** [comment]
This is great. Many thanks :)

**Comment by ReinerBforartists** [comment]
Resurrecting this thread. It's time to spread the word again, now that we have a stable version of Bforartists 2

**Comment by ReinerBforartists** [comment]
Announced Bforartists at the usual places. See first posting.
Plus
https://www.psd-tutorials.de/forum/threads/bforartists-2-erschienen.182158/#post-2724522
And 
https://cgpress.org/archives/bforartists-2-released.html

**Comment by ReinerBforartists** [comment]
Not my favourite job. But we have a few more announcements now. The good thing is, the fork is known enough that it spreads by itself now. It was not necessary to post it at gamedev.net again for example. And also other pages have picked up the release.

Still waiting if the german magazine digitalproduktion will write an article about Bforartists 2 again. I was in contact with them. But i guess this time it will be much shorter, if even.

That should be enough for now. Unassigning. On with other things.

**Comment by ReinerBforartists** [comment]
Ah and when somebody finds the time to write a wikipedia article ... XD

**Comment by ReinerBforartists** [comment]
I'm moving this one out of the tracker since it is no task to solve really. But fits perfectly into the discussion.

**Comment by ReinerBforartists** [comment]
Contacted Softpedia to update the screenshots. They are still from Bforartists 1 ...

https://www.softpedia.com/get/Multimedia/Graphic/Graphic-Editors/Bforartists.shtml

**Comment by ReinerBforartists** [comment]
Added a reddit channel for Bforartists.

Somebody who wants to become mod there? Draise?

https://www.reddit.com/r/Bforartists/comments/oqkqrl/welcome_to_the_new_bforartists_reddit_channel/

**Comment by ReinerBforartists** [comment]
Geez. Seems that making an official community at reddit is not allowed.

> Association to a Brand:
> We love that so many of you want to talk about brands and provide a forum for discussion. Remember to always flag your community as “unofficial” and be clear in your community description that you don’t actually represent that brand.

**Comment by ReinerBforartists** [comment]
Contacted reddit admins. Let's have a look what the outcome is.

> Hello,
> 
> I am having problems understanding some guidelines here, and need some assistance in understanding the legal terms of a community subreddit.
> 
> I am a hobby devleoper. And work since over six years for free at a fork of the popular 3d software Blender. 
> 
> Blender has a community subreddit here. https://www.reddit.com/r/blender/
> 
> And so i thought i create one for Bforartists since i would love to share my progress and news and thoughts around Bforartists and its relationship to Blender. Which i have now.
> 
> It is located here: https://www.reddit.com/r/Bforartists
> 
> Bforartists is completely non commercial and purely hobby driven. It is FOSS software like Blender. And there is, different to Blender, no money involved at any point. It is no business. And the software is free.
> 
> https://www.bforartists.de/
> 
> However, while reading through the community guidelines i stumbled across this passus here:
> 
> ""
> Association to a Brand:
> We love that so many of you want to talk about brands and provide a forum for discussion. Remember to always flag your community as “unofficial” and be clear in your community description that you don’t actually represent that brand.
> ""
> 
> Well, i am of course associated with Bforartists, i am the developer behind it. And so it would also be an official channel then. But i am a hobbyist. Not a company. Even when i have created a brand.
> 
> I am confused. Does this mean that i am not allowed to create a community subreddit for it? But Blender can? Shall i remove my community channel again? I don't want to violate any rule here. 
> 
> Could you please help me understand this passus? 
> 
> Kind regards
> Reiner from Bforartists
>

**Comment by ReinerBforartists** [comment]
Instagram has banned me for registering already. I was never able to log in. But found a message at the login page today that i violated the Instagram rules. And the account was therefore deleted.

Precrime at work ...

**Comment by ReinerBforartists** [comment]
Never heard from Instagram again, and most probably never will. 

But Reddit wrote back

> Hey there!
> 
> Thanks for reaching out. You're more than welcome to have created r/Bforartists—thanks for creating a community for others to join!
> 
> You don't need to mark it unofficial but it'll be good to be transparent with the redditors in the community that you work on the software itself.
> 
> Hope that helps—take care for now!

**Comment by ReinerBforartists** [comment]
I have registered to some public facebook 3d groups. This is a good marketing weapon as i just have learned. Of course just when you post something over there. 

@Draise, i have also registered to some in the name of Bforartists. Maybe you can share some of your posts in this groups. Of course just in case it fits into their scope. Spamming a facebook group with promotional content can do the opposite of good. So we have to do it decent.

https://www.facebook.com/Artistsofworld/
https://www.facebook.com/groups/cg3dworld
https://www.facebook.com/groups/511035342752825
https://www.facebook.com/groups/ce3dmodeling
https://www.facebook.com/groups/3dmodelingandanimation
https://www.facebook.com/groups/290530545401744
